### ISSUE TITLE
Typo in Category menu link; now getting 404 page

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#255]
I thought this was fine after the initial migration -- but Janelle posted in the channel that there's a typo on the UX drop down menu at Accessibility; it is now missing an s in the URL

In the menu, can one of you pls change https://www.digitalgov.gov/categories/accesibility/ 
to https://www.digitalgov.gov/categories/accessibility/ 

thx!

**Comment by jeremyzilar** [comment]
This has been completed 🎉👾
